js/scoreboard.js

```diff
@@ -0,0 +1,7 @@
+var ScoreBoard = {
```

Comment:
you forgot your comments to this obj literal

Reply:
you bitch

Reply:
ey putilla, esto esta listo